components/collective-page/BudgetStats.js

```diff
@@ -164,7 +166,7 @@ const BudgetStats = ({ collective, stats, horizontal }) => {
               currency={collective.currency}
             />
           </StatContainer>
-          {!isFund && (
+          {!isFund && stats.totalAmountReceived?.valueInCents && stats.yearlyBudget?.valueInCents && (
```

Comment:
Make the "yearly budget balance" section optional in case `totalAmountReceived` or `yearlyBudget` are not returned (for whatever reason).

Reply:
What about Monthly recurring?

Reply:
That was addressed.